odoo/addons/base/models/ir_ui_view.py

```diff
@@ -431,13 +431,34 @@ def _valid_inheritance(self, arch):
                         self._raise_view_error(message, node)
         return True
 
+    def _validate_render_report(self):
+        for view in self:
+            if view.type != 'qweb':
+                return
+            arch_value = 't-call="%s"' % view.xml_id
```

Comment:
a view could use single quote in its t-call as well as double quotes, you should probably use the `_` postgres wildcard
```suggestion
            arch_value = 't-call=_%s_' % view.xml_id
```